go.mod

```diff
@@ -76,3 +76,7 @@ require (
 )
 
 replace github.com/onsi/gomega => github.com/onsi/gomega v1.16.0
+
+// builds are broken with selinux v1.10.1, see https://github.com/containers/buildah/pull/3875
+// once PR is merged and vendored into podman we have to remove this
+replace github.com/opencontainers/selinux => github.com/opencontainers/selinux v1.10.0
```

Comment:
This is already merged https://github.com/containers/buildah/pull/3875 should we remove this ? However I think a vendor dance would be needed.

Reply:
https://github.com/containers/podman/pull/13733 will vendor in a new buildah version
Updating buildah in this PR is out of scope IMO